src/hero/CodeTemplates.js

```diff
@@ -0,0 +1,14 @@
+import {CardReactTemplate} from '../CardCatalog';
+import {TopAppBarReactTemplate} from '../TopAppBarCatalog';
+import {ButtonReactTemplate} from '../ButtonCatalog';
+import {TextFieldReactTemplate} from '../TextFieldCatalog';
+
+const ReactTemplates = {
+  'ButtonHero': (config) => ButtonReactTemplate(config),
+  'CardHero': (config) => CardReactTemplate(config),
+  'CheckboxHero': () => {},
```

Comment:
Why is this here?

Reply:
we were also going to group in checkbox to the updated heros. Will remove.